rental.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+
+  def book=(book)
+    @book = book
+    book.rentals.push(self) unless book.rental.include?(self)
+  end
+
+  def person=(person)
+    @person = person
+    person.rentals.push(self) unless person.rental.include?(self)
+  end
```

Comment:
[OPTIONAL] 

- You could improve this by reducing the extra methods since you have them in the parameter of the constructor above like this below:

```ruby
 def initialize(date, book, person)
    @date = date
    @person = person
    @book = book
    person.rentals << self
    book.rentals << self
  end